src/neuroconv/tools/roiextractors/roiextractors.py

```diff
@@ -1042,14 +1098,12 @@ def add_segmentation(
     # Add device:
     add_devices(nwbfile=nwbfile, metadata=metadata)
 
-    # Add imaging plane
-    add_imaging_plane(nwbfile=nwbfile, metadata=metadata)
-
-    # PlaneSegmentation:
```

Comment:
The reason why this is removed is because `add_imaging_plane()` is also called from `add_plane_segmentation()`. If you feel like this change is not justified I can revert